src/node_dotenv.cc

```diff
@@ -107,13 +107,28 @@ Local<Object> Dotenv::ToObject(Environment* env) const {
 
 std::string_view trim_spaces(std::string_view input) {
   if (input.empty()) return "";
-  if (input.front() == ' ') {
-    input.remove_prefix(input.find_first_not_of(' '));
+
+  size_t input_size = input.size();
+
+  size_t pos_start = 0;
+  while (pos_start < input_size) {
+    const char ch = input.at(pos_start);
+    if (ch != ' ' && ch != '\t' && ch != '\n') {
+      break;
+    };
+    pos_start++;
   }
```

Comment:
You can do the same thing for other while loop as well:

```suggestion
  auto pos_start = input.find_first_not_of(" \t\n");
  if (pos_start == std::string_view::npos) {
    input.clear();
    return;
  }
```